/**:
  ros__parameters:
    map_length: 100.0     # [m]
    map_resolution: 0.5 # [m]

    use_height_filter: true
    enable_single_frame_mode: false
    # use sensor pointcloud to filter obstacle pointcloud
    filter_obstacle_pointcloud_by_raw_pointcloud: false

    # grid map coordinate
    map_frame: "map"
    base_link_frame: "base_link"
    # center of the grid map
    gridmap_origin_frame: "base_link"
    # ray-tracing center: main sensor frame is preferable like: "velodyne_top"
    scan_origin_frame: "base_link"
